modules/bot.py

```diff
@@ -215,7 +261,12 @@ def _teams_picking_to_str(self):
         if self.ranked:
             unpicked_str = "[" + ", ".join(["`{0}{1}`".format(utils.rating_to_icon(self.ranks[i.id]), (i.nick or i.name).replace("`","")) for i in sorted(self.unpicked, key=lambda p: self.ranks[p.id], reverse=True)]) + "]"
         else:
-            unpicked_str = "[" + ", ".join(["`{0}`".format((i.nick or i.name).replace("`","")) for i in self.unpicked]) + "]"
+            player_strs = []
+            for position, player in self.unpicked_pool.all.items():
+                player_strs.append(
+                    "`{0}. {1}".format(position, (player.nick or player.name).replace("`",""))
```

Comment:
This is where we're now creating a message that contains the position of that player and the player's name/nick.

e.g. a single player_str might be
```python
"1. nolja"
```